src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitRadioInput/index.js

```diff
@@ -12,7 +12,6 @@ const IncidentSplitRadioInput = ({ id, name, display, options, initialValue, reg
 
   const onChange = useCallback(
     event => {
-      event.preventDefault();
```

Comment:
This small change fixes https://datapunt.atlassian.net/browse/SIG-3300. Because the event was unnecessarily prevented, react-hook-form would not pick up the change (unless other form elements were changed afterwards).